src/scripts/migrate.py

```diff
@@ -57,7 +57,7 @@ def _valid_parent_conditions(
 
     For a term to be immediately migratable, it must either (a) have no parents, or (b) all of its parents must be
     mapped, obsolete, or excluded"""
-    return not parents or [any([x in y for y in [mapped, excluded, obsolete]]) for x in parents]
+    return not parents or all([(x in mapped) or (x in excluded) or (x in obsolete) for x in parents])
```

Comment:
[Bugfix](https://github.com/monarch-initiative/mondo-ingest/pull/316#discussion_r1208112732): Corrected parents condition and simplified expression in _valid_parent_conditions()